unit-test/cs_compute_tests.c

```diff
@@ -841,7 +857,54 @@ void CS_ComputeApp_Test_Nominal(void)
 
     /* Sets AppInfo.CodeSize = 5, sets AppInfo.CodeAddress = 1, AppInfo.AddressesAreValid = true, and returns
      * CFE_SUCCESS */
-    UT_SetHandlerFunction(UT_KEY(CFE_ES_GetAppInfo), CS_COMPUTE_TEST_CFE_ES_GetAppInfoHandler1, NULL);
+    UT_SetHandlerFunction(UT_KEY(CFE_ES_GetModuleInfo), CS_COMPUTE_TEST_CFE_ES_GetModuleInfoHandler1, NULL);
+
+    /* Set to fail condition "NewChecksumValue != ResultsEntry -> ComparisonValue" */
+    UT_SetDeferredRetcode(UT_KEY(CFE_ES_CalculateCRC), 1, 2);
+
+    /* Execute the function being tested */
+    Result = CS_ComputeApp(&ResultsEntry, &ComputedCSValue, &DoneWithEntry);
+
+    /* Verify results */
+    UtAssert_True(DoneWithEntry == true, "DoneWithEntry == true");
+
+    UtAssert_True(ResultsEntry.NumBytesToChecksum == 5, "ResultsEntry.NumBytesToChecksum == 5");
+    UtAssert_True(ResultsEntry.StartAddress == 1, "ResultsEntry.StartAddress == 1");
+
+    UtAssert_True(ComputedCSValue == 2, "ComputedCSValue == 2");
+    UtAssert_True(ResultsEntry.ByteOffset == 0, "ResultsEntry.ByteOffset == 0");
+    UtAssert_True(ResultsEntry.TempChecksumValue == 0, "ResultsEntry.TempChecksumValue == 0");
+
+    UtAssert_True(Result == CFE_SUCCESS, "Result == CFE_SUCCESS");
+
+    call_count_CFE_EVS_SendEvent = UT_GetStubCount(UT_KEY(CFE_EVS_SendEvent));
+
+    UtAssert_True(call_count_CFE_EVS_SendEvent == 0, "CFE_EVS_SendEvent was called %u time(s), expected 0",
+                  call_count_CFE_EVS_SendEvent);
```

Comment:
UtAssert_STUB_COUNT(CFE_EVS_SendEvent, 0); is the preferred pattern.  Apply everywhere instead of using UT_GetStubCount.

Reply:
Done! Let me know if I missed anything.